---
title: "ggtree分析群落多样性数据"
author: "Gao, Chun-Hui"
date: "2018年4月3日"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
## set root dir
knitr::opts_knit$set(root.dir = "H:/Spring_Work/35. Data/2018-4-3 ggtree")
library(ggplot2)
library(ggtree)
library(treeio)
```

# 数据来源

## 原始数据

原始数据分为两个部分。

- OTU序列，fasta格式。
- OTU在各个样品中的丰度，即OTU Table，文本格式。

## OTU分类和进化树构建

OTU分类通过Silva Aligner <https://www.arb-silva.de/aligner> 获得，该程序同时可以获得Tree。

> Taxonomic classification and phylogenetic tree esitimation were performed using Silva Aligner (version 1.2.11). The parameter of minmum identity with query sequence in taxonomic classification were set to 0.90; The RAxML program is used to build a maximum likelihood phylogenetic tree; and other parameters are in defaults. 

分析完成后下载分类文件和进化树文件。

## MEGA6构建进化树

我也使用MEGA6构建了一个进化树。方法不做赘述。

但是Silva Aligner在构建进化树时，可以对16S rRNA基因可变区域设置遮罩，显然更具有可行性。因此，最终使用了Aligner的进化树。


# 数据清洗

## Tree

ggtree支持读入nwk等树格式，读取文件之后，主要修改`tree$tip.label`。修改后重新保存为`otu.tree`文件。

```{r tree_cleanup}
## 这是之前使用的MEGA6 Tree
# tree <- read.tree(file="otu_reps-isolates.nwk")
## 这是现在使用的RAxML Tree
tree <- read.tree(file="arb-silva.de_2018-05-14_id525911.tree")
tree$tip.label <- gsub("(OTU\\d+)_.*","\\1",tree$tip.label,perl=TRUE)
tree$tip.label <- gsub("^.*?_","",tree$tip.label)
tree$tip.label <- gsub("_"," ",tree$tip.label)
tree$species <- tree$tip.label
tree$tip.label <- gsub(".*sp\\.\\s","",tree$tip.label,perl=TRUE)

write.tree(tree,file = "otu.tree")
```


## 分类信息

分类信息采用Silva结果。需要将Sequence identifier统一，并将分类划分到界门纲目科属种等不同分类水平。

```{r taxonomic_classification_cleanup}
## 读取分类信息
tax <- read.delim(file="arb-silva.de_align_resultlist_525911.csv",sep = ";")
tax <- tax[,c("sequence_identifier","lca_tax_slv")]

## 这个原始序列的ID格式为：OTU1_D1_213891，现在去掉后面的内容，仅留下唯一OTUID
tax$sequence_identifier <- gsub("(OTU\\d+)_.*","\\1",tax$sequence_identifier)

## 各个分类水平（为了是分类水平的唯一化，不宜直接使用各水平的名字。
## 例如不同门可能都会有unclassifed这一词语，但是不同门直接的unclassified显然是不一样的。
## 因此，如果要到某一水平，那么将其上游的分类也应当包括在内。
## 门水平
tax$phylum <- sapply(tax$lca_tax_slv, function(x) paste(unlist(strsplit(as.character(x),";"))[1:2],collapse = ";"))
## 纲水平
tax$class <- sapply(tax$lca_tax_slv, function(x) paste(unlist(strsplit(as.character(x),";"))[1:3],collapse = ";"))
## 属水平
tax$genus <- sapply(tax$lca_tax_slv, function(x) paste(unlist(strsplit(as.character(x),";"))[1:6],collapse = ";"))

rownames(tax) <- tax$sequence_identifier

tax <- tax[,c("phylum","class","genus")]
## 分类信息保存为新文件
write.csv(tax, file = "otu_tax.csv")

## 前几行的内容是
head(tax)

```


## OTU Table

```{r otu_table_cleanup}
# 读取OTU表
otu_table <- read.delim("otu_table.txt",row.names = "OTU",header = T)

# 去除不含分类信息的行
# idx <- which(rownames(otu_table) %in% unclassified_otu)
# otu_table <- otu_table[-idx,]

# 标准化：将count转变为百分比
norm <- t(t(otu_table)/colSums(otu_table,na=T))

## OTU丰度事实上差异很大
hist(log10(apply(norm, 1, var)),xlab="log10(variance)", breaks=100,main="A large fraction of OTUs have very low variance")

write.csv(norm,file = "norm_otu_table.csv")

```

# 作图

```{r ggtree}
## Data
tree <- read.tree("otu.tree")
tax <- read.csv("otu_tax.csv",row.names = 1)
otu_table <- read.csv("norm_otu_table.csv",row.names = 1)


## groupOTU
groupinfo <- split(rownames(tax),tax$genus)
tree <- groupOTU(tree,groupinfo)

## 按照tax的genus水平统一otu_table
tax <- tax[rownames(tax) %in% rownames(otu_table),]
identical(rownames(tax), rownames(otu_table))
data <- cbind(tax,otu_table)

tax_by_genus <- data.frame(OTU=rownames(tax),genus=tax$genus)
sum_by_genus <- data %>% group_by(genus) %>% summarise_at(colnames(data)[4:11],sum)

data <- merge(tax_by_genus,sum_by_genus,by="genus")
rownames(data) <- data$OTU
data <- data[,3:10]

## 作图
p <- ggtree(tree,aes(color=group)) + theme(legend.position = "none")
gheatmap(p,log10(data)) + theme(legend.position = "none")


```

# 树的OTU着色问题



The group 0 is reserved for any branches that are not belonging to any groups.

For legend, you can rename it or disable it.

Try :

```{r eval=FALSE, include=FALSE}
ggtree(tree, aes(color=group)) + geom_tiplab() +
    scale_color_manual(values=c('black', rainbow_hcl(4)), breaks=names(cls)) + theme(legend.position="right")
```

It will display the legend without ‘group 0’.


```{r}
# 上面的树着色会有问题
# 使用tax中的id则会在ggtree中多一个0出来，分类和图都错了
g1 <- split(tax$full_name,tax$phylum)
ggtree(groupOTU(tree,g1),aes(color=group))+geom_tiplab()+theme(legend.position = "right")+xlim(0,0.5)

# 这里使用tree tip.label作为第一个参数则分类个数正常，但是分类错误
g2 <- split(tree$tip.label,tax$phylum)
ggtree(groupOTU(tree,g2),aes(color=group))+geom_tiplab()+theme(legend.position = "right")+xlim(0,0.5)
```

```{r random tree}
## 随机树按分类着色
# 生成一个随机树和分类信息
require(ggtree)
# random tree of OTUs
set.seed(0)
tr <- rtree(100)

# taxnomy of the OTUs
tax <- data.frame(name = sample(tr$tip.label),
                  phylum = sapply(sample(1:3,Ntip(tr),replace = TRUE), function(x) paste("phylum",x,sep="_")),
                  class = sapply(sample(4:6,Ntip(tr),replace = TRUE), function(x) paste("class",x,sep="_")),
                  family = sapply(sample(7:12,Ntip(tr),replace = TRUE), function(x) paste("family",x,sep="_")),
                  genus = sapply(sample(13:20,Ntip(tr),replace = TRUE), function(x) paste("genus",x,sep="_"))
                  )

# species abundance of the OTUs in samples
data <- data.frame(matrix(runif(600),ncol=6),row.names = sample(tr$tip.label))

p <- ggtree(tr) + geom_tiplab(align = TRUE)

gheatmap(p,data)

library(dplyr)
by_genus <- group_by(cbind(tax,data),phylum,class,genus)
a <- summarize(by_genus,a1 = mean(X1), a2 = mean(X2), a3 = mean(X3), a4 = mean(X4), a5 = mean(X5), a6=mean(X6), otu = paste0(name,collapse = "-"))


```


## ggtree安装
```
BiocInstaller::biocLite("ggtree")
```

