<!DOCTYPE html>
<html><head>
		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
		<title>Zotero Report</title>
		<link rel="stylesheet" type="text/css" href="Zotero%20Report_files/detail.css">
		<link rel="stylesheet" type="text/css" media="screen,projection" href="Zotero%20Report_files/detail_screen.css">
		<link rel="stylesheet" type="text/css" media="print" href="Zotero%20Report_files/detail_print.css">
	</head>
	<body>
		<ul class="report combineChildItems">
			<li id="item_86B8P5T7" class="item journalArticle">
			<h2>大肠杆菌表面感应机制研究进展</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>立亮 王</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>春辉 高</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>一超 吴</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>巧云 黄</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>鹏 蔡</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="http://kns.cnki.net/KCMS/detail/detail.aspx?dbcode=CJFQ&amp;dbname=CJFDTEMP&amp;filename=ZJNY201706004&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVWI2cE1qQmM1ZStoSm4yMFVDWFBxMkphRT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4ggI8Fm4gTkoUKaID8j8gFw%21%21&amp;v=MTkzNTkxRnJDVVJMS2ZaZVJuRnluaFVyN0JQeWZGZDdHNEg5Yk1xWTlGWUlSOGVYMUx1eFlTN0RoMVQzcVRyV00=">http://kns.cnki.net/KCMS/detail/detail.aspx?dbcode=CJFQ&amp;dbname=CJFDTEMP&amp;filename=ZJNY201706004&amp;uid=WEEvREcwSlJHSldRa1FhdkJkVWI2cE1qQmM1ZStoSm4yMFVDWFBxMkphRT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4ggI8Fm4gTkoUKaID8j8gFw!!&amp;v=MTkzNTkxRnJDVVJMS2ZaZVJuRnluaFVyN0JQeWZGZDdHNEg5Yk1xWTlGWUlSOGVYMUx1eFlTN0RoMVQzcVRyV00=</a></td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>06</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>685-690</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>浙江大学学报(农业与生命科学版)</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>1008-9209</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2017</td>
					</tr>
					<tr>
					<th>Call Number</th>
						<td>33-1247/S</td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>2018/2/28 上午10:50:22</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>CNKI</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>中文;</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>大肠杆菌是人和动物胃肠道的主要兼性厌氧菌群,既能在宿主体内生存,也能吸附在环境中的固体介质表面形成生物膜而得以存活和传播。吸
附是生物膜形成的关键,它能改变细胞运动性和代谢等相关基因的表达,进而影响细胞的性质和行为。细菌对表面的"感知"以及初始吸附后细胞的"响应"过程统
称为"表面感应"。对细菌表面感应机制的研究有助于我们加深对微生物尤其是病原菌在土壤环境中的存活和迁移,以及土壤生物膜形成过程的理解。本文结合多种
微生物表面感应的研究进展,阐述了表面感应的概念,重点关注了近期有关大肠杆菌表面感应的研究,并对未来工作进行了展望。</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2018/2/28 上午10:50:22</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2018/7/19 上午9:47:11</td>
					</tr>
				</tbody></table>
				<h3 class="tags">Tags:</h3>
				<ul class="tags">
					<li>课题6研究论文</li>
					<li>重点研发计划第一标注</li>
				</ul>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_559YZNJT">王 et al_2017_大肠杆菌表面感应机制研究进展.pdf					</li>
				</ul>
			</li>


			<li id="item_4XI6U6EA" class="item journalArticle">
			<h2>自然环境中的多物种生物膜:研究方法及社群相互作用</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>晓洁 孙</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>春辉 高</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>巧云 黄</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>鹏 蔡</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="http://www.cnki.net/KCMS/detail/detail.aspx?QueryID=3&amp;CurRec=7&amp;filename=NHFZ201701002&amp;dbname=CJFDPREP&amp;dbcode=CJFQ&amp;pr=&amp;urlid=&amp;yx=&amp;uid=WEEvREcwSlJHSldRa1Fhb09jMjQwK09lNmhyeE1KZ1h5cEU1MXNmUjFETT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4ggI8Fm4gTkoUKaID8j8gFw%21%21&amp;v=MjA2MTZyV00xRnJDVVJMMmZadVpuRnluaFc3L0FLeVhOZExHNEg5Yk1ybzlGWm9SOGVYMUx1eFlTN0RoMVQzcVQ=">http://www.cnki.net/KCMS/detail/detail.aspx?QueryID=3&amp;CurRec=7&amp;filename=NHFZ201701002&amp;dbname=CJFDPREP&amp;dbcode=CJFQ&amp;pr=&amp;urlid=&amp;yx=&amp;uid=WEEvREcwSlJHSldRa1Fhb09jMjQwK09lNmhyeE1KZ1h5cEU1MXNmUjFETT0=$9A4hF_YAuvQ5obgVAqNKPCYcEjKensW4ggI8Fm4gTkoUKaID8j8gFw!!&amp;v=MjA2MTZyV00xRnJDVVJMMmZadVpuRnluaFc3L0FLeVhOZExHNEg5Yk1ybzlGWm9SOGVYMUx1eFlTN0RoMVQzcVQ=</a></td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>01</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>6-14</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>农业资源与环境学报</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2017</td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>2017/1/8 上午10:59:29</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>CNKI</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>自然界中的微生物大多以多物种生物膜的形式存在,这种生命形式可以增加微生物对外界环境胁迫的耐受性。在多物种生物膜中,微生物之间
的相互作用包括合作、竞争、信号分子的传递、以及水平基因转移等。研究自然环境中多物种生物膜内部微生物之间的相互作用,不仅对于理解自然生物膜的形成和
演替过程具有重要意义,而且也可以为多物种生物膜在环境质量提升、微生物多样性维持等方面提供理论指导。本文主要分为两个部分:第一部分总结了环境中多物
种生物膜的研究方法,包括实验室模拟体系下生物膜的构建及其分析检测技术;第二部分论述了天然多物种生物膜内细菌间的相互作用。</td>
					</tr>
					<tr>
					<th>Short Title</th>
						<td>自然环境中的多物种生物膜</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2017/1/8 上午10:59:29</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2018/7/19 上午9:44:35</td>
					</tr>
				</tbody></table>
				<h3 class="tags">Tags:</h3>
				<ul class="tags">
					<li>课题6研究论文</li>
					<li>重点研发计划第一标注</li>
				</ul>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_UX2ZXJJA">孙晓 et al_2017.pdf					</li>
				</ul>
			</li>


			<li id="item_3RSB5RTQ" class="item journalArticle">
			<h2>A Novel marRAB Operon Contributes to the Rifampicin Resistance in Mycobacterium smegmatis</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Haiwei Zhang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Long Gao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Jiaoling Zhang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Weihui Li</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Min Yang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Hua Zhang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chunhui Gao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Zheng-Guo He</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="http://dx.doi.org/10.1371/journal.pone.0106016">http://dx.doi.org/10.1371/journal.pone.0106016</a></td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>9</td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>8</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>e106016</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>PLoS ONE</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>八月 25, 2014</td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>PLoS ONE</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1371/journal.pone.0106016">10.1371/journal.pone.0106016</a></td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>2014/12/16 上午9:42:15</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>PLoS Journals</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>The multiple-antibiotic resistance regulator (MarR) plays an 
important role in modulating bacterial antibiotic resistance. However, 
the regulatory model of the marRAB operon in mycobacteria remains to be 
characterized. Here we report that a MarR, encoded by Ms6508, and its 
marRAB operon specifically contribute to rifampicin (RIF) resistance in 
Mycobacterium smegmatis. We show that the MarR recognizes a conserved 
21-bp palindromic motif and negatively regulates the expression of two 
ABC transporters in the operon, encoded by Ms6509–6510. Unlike other 
known drug efflux pumps, overexpression of these two ABC transporters 
unexpectedly increased RIF sensitivity and deletion of these two genes 
increased mycobacterial resistance to the antibiotic. No change can be 
detected for the sensitivity of recombinant mycobacterial strains to 
three other anti-TB drugs. Furthermore, HPLC experiments suggested that 
Ms6509–Ms6510 could pump RIF into the mycobacterial cells. These 
findings indicated that the mycobacterial MarR functions as a repressor 
and constitutively inhibits the expression of the marRAB operon, which 
specifically contributes to RIF resistance in M. smegmatis. Therefore, 
our data suggest a new regulatory mechanism of RIF resistance and also 
provide the new insight into the regulatory model of a marRAB operon in 
mycobacteria.</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2014/12/16 上午9:42:15</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2014/12/16 上午9:42:15</td>
					</tr>
				</tbody></table>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_SJ863EVT">PLoS ONE_2014_A Novel marRAB Operon Contributes to the Rifampicin Resistance in Mycobacterium smegmatis.pdf					</li>
				</ul>
			</li>


			<li id="item_XDINK7JG" class="item journalArticle">
			<h2>A TetR-like regulator broadly affects the expressions of diverse genes in Mycobacterium smegmatis</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Min Yang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chunhui Gao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Tao Cui</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Jingning An</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Zheng-Guo He</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="http://nar.oxfordjournals.org/content/40/3/1009">http://nar.oxfordjournals.org/content/40/3/1009</a></td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>40</td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>3</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>1009-1020</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>Nucleic Acids Research</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>0305-1048, 1362-4962</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>02/01/2012</td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>PMID: 21976733</td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>Nucl. Acids Res.</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1093/nar/gkr830">10.1093/nar/gkr830</a></td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>2016/3/3 下午4:08:20</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>nar.oxfordjournals.org</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>en</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>Transcriptional regulation plays a critical role in the life 
cycle of Mycobacterium smegmatis and its related species, M. 
tuberculosis, the causative microbe for tuberculosis. However, the key 
transcriptional factors involved in broad regulation of diverse genes 
remain to be characterized in mycobacteria. In the present study, a 
TetR-like family transcriptional factor, Ms6564, was characterized in M.
 smegmatis as a master regulator. A conserved 19 bp-palindromic motif 
was identified for Ms6564 binding using DNaseI footprinting and EMSA. A 
total of 339 potential target genes for Ms6564 were further 
characterized by searching the M. smegmatis genome based on the sequence
 motif. Notably, Ms6564 bound with the promoters of 37 cell cycle and 
DNA damage/repair genes and regulated positively their expressions. The 
Ms6564-overexpressed recombinant strain yielded 5-fold lower mutation 
rates and mutation frequencies, whereas deletion of Ms6564 resulted in 
∼5-fold higher mutation rates for the mutant strain compared with the 
wild-type strain. These findings suggested that Ms6564 may function as a
 global regulator and might be a sensor necessary for activation of DNA 
damage/repair genes.</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2016/3/3 下午4:08:20</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2016/3/3 下午4:08:20</td>
					</tr>
				</tbody></table>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_JN4UQDAG">Nucl. Acids Res._2012_A TetR-like regulator broadly affects the expressions of diverse genes in Mycobacterium smegmatis.pdf					</li>
					<li id="item_BWWXTPZ4">PubMed entry					</li>
				</ul>
			</li>


			<li id="item_CCENUPNV" class="item journalArticle">
			<h2>An ArsR-like transcriptional factor recognizes a conserved 
sequence motif and positively regulates the expression of phoP in 
mycobacteria</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chun-Hui Gao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Min Yang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Zheng-Guo He</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/21782791">http://www.ncbi.nlm.nih.gov/pubmed/21782791</a></td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>411</td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>4</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>726-731</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>Biochemical and Biophysical Research Communications</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>1090-2104</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>Aug 12, 2011</td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>00013 
PMID: 21782791</td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>Biochem. Biophys. Res. Commun</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1016/j.bbrc.2011.07.014">10.1016/j.bbrc.2011.07.014</a></td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>2011/8/31 上午10:51:05</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>NCBI PubMed</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>Transcriptional regulation plays a critical role during the 
infection of Mycobacterium tuberculosis, the causative agent of 
tuberculosis. A two-component system, PhoPR, is clearly involved in the 
regulation of pathogenic virulence and persistence. However, the 
regulatory mechanism, as well as the regulator, of the phoPR operon 
remains uncharacterized in M. tuberculosis and its related species thus 
far. In the present study, we characterize an ArsR transcriptional 
factor, corresponding to Rv2034 and Ms6762 in M. tuberculosis and 
Mycobacterium smegmatis, respectively, as the first regulator of phoP in
 both mycobacterial species. The interaction between ArsR regulator and 
target promoters is conserved in these two mycobacterial species, and an
 inverted repeat sequence motif is successfully mapped out for the 
recognition of ArsR. Utilizing lacZ reporter genes and overexpression 
analysis, the ArsR regulator is shown to positively regulate the 
expression of phoP in M. smegmatis, different from most ArsR family 
regulators generally as a repressor. The current study establishes a 
direct link between the ArsR transcriptional factor and the regulation 
of phoP in mycobacteria. Our findings imply that ArsR may be involved in
 the pathogenesis of M. tuberculosis through its regulation of the phoPR
 operon.</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2011/8/31 上午10:51:05</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2015/6/5 下午6:09:05</td>
					</tr>
				</tbody></table>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_BNUIBH93">Biochem. Biophys. Res. Commun_2011_An 
ArsR-like transcriptional factor recognizes a conserved sequence motif 
and positively regulates the expression of phoP in mycobacteria.pdf					</li>
				</ul>
			</li>


			<li id="item_8XBK4VFD" class="item journalArticle">
			<h2>Bacillus subtilis biofilm development in the presence of soil clay minerals and iron oxides</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Wenting Ma</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Donghai Peng</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Sharon L. Walker</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Bin Cao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chun-Hui Gao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Qiaoyun Huang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Peng Cai</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="http://www.nature.com/articles/s41522-017-0013-6">http://www.nature.com/articles/s41522-017-0013-6</a></td>
					</tr>
					<tr>
					<th>Rights</th>
						<td>2017 The Author(s)</td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>3</td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>1</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>4</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>npj Biofilms and Microbiomes</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>2055-5008</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2017/02/09</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1038/s41522-017-0013-6">10.1038/s41522-017-0013-6</a></td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>2017/2/10 上午9:21:59</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>www.nature.com</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>En</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>Clay minerals and metal oxides, as important parts of the soil
 matrix, play crucial roles in the development of microbial communities.
 However, the mechanism underlying such a process, particularly on the 
formation of soil biofilm, remains poorly understood. Here, we 
investigated the effects of montmorillonite, kaolinite, and goethite on 
the biofilm formation of the representative soil bacteria Bacillus 
subtilis. The bacterial biofilm formation in goethite was found to be 
impaired in the initial 24 h but burst at 48 h in the liquid–air 
interface. Confocal laser scanning microscopy showed that the biofilm 
biomass in goethite was 3–16 times that of the control, montmorillonite,
 and kaolinite at 48 h. Live/Dead staining showed that cells had the 
highest death rate of 60% after 4 h of contact with goethite, followed 
by kaolinite and montmorillonite. Atomic force microscopy showed that 
the interaction between goethite and bacteria may injure bacterial cells
 by puncturing cell wall, leading to the swarming of bacteria toward the
 liquid–air interface. Additionally, the expressions of abrB and sinR, 
key players in regulating the biofilm formation, were upregulated at 
24 h and downregulated at 48 h in goethite, indicating the initial 
adaptation of the cells to minerals. A model was proposed to describe 
the effects of goethite on the biofilm formation. Our findings may 
facilitate a better understanding of the roles of soil clays in biofilm 
development and the manipulation of bacterial compositions through 
controlling the biofilm in soils. The effect of three soil minerals on 
biofilm production is clarified by research using the common soil 
bacterium Bacillus subtilis. The mineral composition of soil is known to
 affect biofilm production, but the mechanisms underpinning minerals’ 
influences have not been well studied. Peng Cai and colleagues at 
Huazhong Agricultural University in China, with co-workers in the United
 States and Singapore, studied Bacillus subtilis growing in the presence
 of the minerals montmorillonite, kaolinite, and goethite. Their results
 suggest the minerals, especially goethite, can encourage biofilm 
formation by promoting the bursting of bacterial cells. The effect of 
goethite was attributed to the size of its grains being generally 
smaller than the bacterial cells. By quantifying the effect of these 
minerals, the research will assist understanding of biofilm formation 
and the growth and persistence of bacterial populations in soils.</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2017/2/10 上午9:21:59</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2017/10/30 下午2:49:00</td>
					</tr>
				</tbody></table>
				<h3 class="tags">Tags:</h3>
				<ul class="tags">
					<li>课题6研究论文</li>
				</ul>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_BP6UIJIH">MaW et al_2017.pdf						<div class="note"><div><p xmlns="http://www.w3.org/1999/xhtml" id="title"><strong>Contents</strong></p><ul xmlns="http://www.w3.org/1999/xhtml" style="list-style-type: none; padding-left:0px" id="toc"><li><a href="zotero://open-pdf/0_BP6UIJIH/1">Introduction</a></li><li style="padding-top:8px"><a href="zotero://open-pdf/0_BP6UIJIH/2">Results</a><ul style="list-style-type: none; padding-left:12px"><li style="padding-top:4px"><a href="zotero://open-pdf/0_BP6UIJIH/2">Montmorillonite, kaolinite, and goethite vary in their effects on B. subtilis biofilm formation</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_BP6UIJIH/2">Mineral-induced cell death contributes to enhanced biofilm formation</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_BP6UIJIH/3">The bacteria sense and response to mineral exposure</a></li></ul></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_BP6UIJIH/4">Discussion</a></li><li style="padding-top:8px"><a href="zotero://open-pdf/0_BP6UIJIH/7">Materials and methods</a><ul style="list-style-type: none; padding-left:12px"><li style="padding-top:4px"><a href="zotero://open-pdf/0_BP6UIJIH/7">Bacterial strains and culture conditions</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_BP6UIJIH/7">Clay minerals and iron oxide</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_BP6UIJIH/7">Assays of biofilm development in suspended minerals</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_BP6UIJIH/7">CLSM and image analysis</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_BP6UIJIH/8">Attenuated total reflectance-Fourier transform infrared spectroscopy (ATR-FTIR) experiments</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_BP6UIJIH/8">Effect of clay minerals on bacterial survival</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_BP6UIJIH/8">Biofilm formation in the presence of dead cells</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_BP6UIJIH/8">Morphology measurements of B. subtilis biofilm on mineral surfaces using AFM</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_BP6UIJIH/8">Quantitative real-time polymerase chain reaction (qRT-PCR)</a></li></ul></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_BP6UIJIH/8">Acknowledgements</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_BP6UIJIH/8">Author contributions</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_BP6UIJIH/8">Competing interests</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_BP6UIJIH/8">ACKNOWLEDGMENTS</a></li></ul></div>
					</div>					</li>
				</ul>
			</li>


			<li id="item_GW8HGZCW" class="item journalArticle">
			<h2>Characterization of a Novel ArsR-Like Regulator Encoded by Rv2034 in Mycobacterium tuberculosis</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chun-Hui Gao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Min Yang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Zheng-Guo He</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="http://www.ncbi.nlm.nih.gov/pubmed/22558408">http://www.ncbi.nlm.nih.gov/pubmed/22558408</a></td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>7</td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>4</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>e36255</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>PloS One</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>1932-6203</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2012</td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>00015 
PMID: 22558408</td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>PLoS ONE</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1371/journal.pone.0036255">10.1371/journal.pone.0036255</a></td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>2012/5/23 上午8:41:04</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>NCBI PubMed</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>The genome of Mycobacterium tuberculosis, the causative agent 
of tuberculosis, encodes a large number of putative transcriptional 
regulators. However, the identity and target genes of only a few of them
 have been clearly identified to date. In a recent study, the ArsR 
family regulator Rv2034 was characterized as a novel positive regulator 
of phoP. In the current study, we characterized the auto-repressive 
capabilities of Rv2034 and identified several residues in the protein 
critical for its DNA binding activities. We also provide evidence that 
Rv2034 forms dimers in vitro. Furthermore, by using DNaseI footprinting 
assays, a palindromic sequence was identified as its binding site. 
Notably, we found that the dosR promoter region contains the binding 
motif for Rv2034, and that Rv2034 positively regulates the expression of
 the dosR gene. The potential roles of Rv2034 in the regulation of lipid
 metabolism and hypoxic adaptation are discussed.</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2012/5/23 上午8:41:04</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2015/6/5 下午6:09:06</td>
					</tr>
				</tbody></table>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_2QNA4N62">PLoS ONE_2012_Characterization of a Novel ArsR-Like Regulator Encoded by Rv2034 in Mycobacterium tuberculosis.pdf					</li>
				</ul>
			</li>


			<li id="item_5WMCVZQI" class="item journalArticle">
			<h2>Characterization of the Interaction and Cross-Regulation of Three Mycobacterium tuberculosis RelBE Modules</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Min Yang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chunhui Gao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Yi Wang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Hua Zhang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Zheng-Guo He</td>
					</tr>
					<tr>
						<th class="editor">Editor</th>
						<td>Ching-Hong Yang</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="http://dx.plos.org/10.1371/journal.pone.0010672">http://dx.plos.org/10.1371/journal.pone.0010672</a></td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>5</td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>5</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>e10672</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>PLoS ONE</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>1932-6203</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2010-5-17</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1371/journal.pone.0010672">10.1371/journal.pone.0010672</a></td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>2015/11/30 上午9:48:32</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>CrossRef</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>en</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2015/11/30 上午9:48:32</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2015/11/30 上午9:48:32</td>
					</tr>
				</tbody></table>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_HPMSXHH5">2010_Characterization of the Interaction and Cross-Regulation of Three Mycobacterium tuberculosis RelBE Modules_2.pdf					</li>
				</ul>
			</li>


			<li id="item_89V6S3BV" class="item journalArticle">
			<h2>Characterization of the interaction between a SirR family 
transcriptional factor of Mycobacterium&nbsp;tuberculosis, encoded by 
Rv2788, and a pair of toxin-antitoxin proteins RelJ/K, encoded by Rv3357
 and Rv3358</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Min Yang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chun-Hui Gao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Jialing Hu</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chao Dong</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Zheng-Guo He</td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>281</td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>12</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>2726-2737</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>The FEBS journal</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>1742-4658</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>Jun 2014</td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>PMID: 24725376</td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>FEBS J.</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1111/febs.12815">10.1111/febs.12815</a></td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>NCBI PubMed</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>eng</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>Toxin-antitoxin (TA) systems play significant roles in the 
regulation of bacterial growth and persistence, and their functions 
usually depend on protein-protein interaction between their constituent 
TA proteins. However, the regulatory mechanisms of these systems, 
particularly their interaction with other cellular components, are still
 poorly understood. This study investigated cross-talk between the TA 
module RelJ/K (Rv3357/Rv3358) and the transcriptional regulator 
staphylococcal iron regulator repressor (SirR, Rv2788) from 
Mycobacterium&nbsp;tuberculosis. We characterized the physical 
interaction of SirR with both RelJ and RelK using bacterial two-hybrid, 
pull-down and co-immunoprecipitation assays. Similarly to RelK, SirR 
regulates the DNA-binding activity of RelJ and alleviates its inhibitory
 effect on the activity of the Rv3357p promoter. Furthermore, SirR may 
replace RelJ to alleviate the inhibitory effect of the toxin RelK on 
bacterial growth. Conversely, both RelJ and RelK competitively inhibit 
the interaction between SirR and their respective promoters. Thus, our 
results show that SirR interacts with a pair of toxin and antitoxin 
proteins, and exhibits antitoxin-like function to neutralize the toxin. 
These findings demonstrate a novel function of the SirR regulator of 
M.&nbsp;tuberculosis as well as a novel mechanism of regulation of TA 
systems.
STRUCTURED DIGITAL ABSTRACT: 
SirR&nbsp;binds&nbsp;to&nbsp;RelJ&nbsp;by&nbsp;pull down&nbsp;(view 
interaction) SirR&nbsp;binds to RelK by pull down (view interaction) 
RelJ&nbsp;physically interacts with RelK by anti bait 
coimmunoprecipitation (1, 2) SirR&nbsp;physically interacts with RelK by
 anti bait coimmunoprecipitation (view interaction) SirR&nbsp;physically
 interacts with RelJ by anti bait coimmunoprecipitation (view 
interaction) RelJ and SirR&nbsp;physically interact by two-hybrid (view 
interaction) SirR and RelK&nbsp;physically interact by two-hybrid (view 
interaction) RelK and RelJ&nbsp;physically interact by two-hybrid (view 
interaction).</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2014/6/24 下午5:33:43</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2014/12/24 上午10:25:03</td>
					</tr>
				</tbody></table>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_VHMKREXU">FEBS J._2014_Characterization of the 
interaction between a SirR family transcriptional factor of 
Mycobacterium tuberculosis, encoded by Rv2788, and a pair of.pdf					</li>
					<li id="item_AFCPFXN6">PubMed entry					</li>
				</ul>
			</li>


			<li id="item_P4YWZB8W" class="item journalArticle">
			<h2>Co-culture of soil biofilm isolates enables the discovery of novel antibiotics</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chun-Hui Gao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Peng Cai</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Zhunjie Li</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Yichao Wu</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Qiaoyun Huang</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="https://www.biorxiv.org/content/early/2018/06/22/353755">https://www.biorxiv.org/content/early/2018/06/22/353755</a></td>
					</tr>
					<tr>
					<th>Rights</th>
						<td>© 2018, Posted by Cold Spring Harbor Laboratory. This 
pre-print is available under a Creative Commons License 
(Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as 
described at http://creativecommons.org/licenses/by-nc/4.0/</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>353755</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>bioRxiv</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2018-06-22</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1101/353755">10.1101/353755</a></td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>2018/12/13 上午10:56:14</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>www.biorxiv.org</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>en</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>Bacterial natural products (NPs) are considered to be a 
promising source of drug discovery. However, the biosynthesis gene 
clusters (BGCs) of NP are not often expressed, making it difficult to 
identify them. Recently, the study of biofilm community showed bacteria 
may gain competitive advantages by the secretion of antibiotics, 
implying a possible way to screen antibiotic by evaluating the social 
behavior of bacteria. In this study, we have described an efficient 
workflow for novel antibiotic discovery by employing the bacterial 
social interaction strategy with biofilm cultivation, co-culture, 
transcriptomic and genomic methods. We showed that a biofilm dominant 
species, i.e. Pseudomonas sp. G7, which was isolated from cultivated 
soil biofilm community, was highly competitive in four-species biofilm 
communities, as the synergistic combinations preferred to exclude this 
strain while the antagonistic combinations did not. Through the analysis
 of transcriptomic changes in four-species co-culture and the complete 
genome of Pseudomonas sp. G7, we finally discovered two novel 
non-ribosomal polypeptide synthetic (NRPS) BGCs, whose products were 
predicted to have seven and six amino acid components, respectively. 
Furthermore, we provide evidence showing that only when Pseudomonas sp. 
G7 was co-cultivated with at least two or three other bacterial species 
can these BGC genes be induced, suggesting that the co-culture of the 
soil biofilm isolates is critical to the discovery of novel antibiotics.
 As a conclusion, we set a model of applying microbial interaction to 
the discovery of new antibiotics.</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2018/12/13 上午10:56:14</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2018/12/13 上午10:56:14</td>
					</tr>
				</tbody></table>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_RRTGAYPR">Gao et al_2018_Co-culture of soil biofilm isolates enables the discovery of novel antibiotics.pdf					</li>
				</ul>
			</li>


			<li id="item_TGYB9SPE" class="item journalArticle">
			<h2>Cross-talk between the three furA orthologs in Mycobacterium smegmatis and the contribution to isoniazid resistance</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chun-Hui Gao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Wen-Ping Wei</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Hui-Ling Tao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Li-Kai Cai</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Wan-Zhong Jia</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Lihua Hu</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Min Yang</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="https://academic.oup.com/jb/advance-article/doi/10.1093/jb/mvz030/5472330">https://academic.oup.com/jb/advance-article/doi/10.1093/jb/mvz030/5472330</a></td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>The Journal of Biochemistry</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2019-5-16</td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>J Biochem</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1093/jb/mvz030">10.1093/jb/mvz030</a></td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>2019/5/28 下午5:03:44</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>academic.oup.com</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>en</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>Abstract.  The ferric uptake regulator A (FurA) plays an 
essential role in responding to oxidative stress in mycobacteria. The 
genome of Mycobacterium smegmatis</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2019/5/28 下午5:03:44</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2019/5/28 下午5:04:47</td>
					</tr>
				</tbody></table>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_W4CTDP5S">Gao et al_Cross-talk between the three furA orthologs in Mycobacterium smegmatis and the.pdf					</li>
					<li id="item_PX3PW4VI">Gao et al_Cross-talk between the three furA orthologs in Mycobacterium smegmatis and the.pdf					</li>
				</ul>
			</li>


			<li id="item_D3Z8N3GW" class="item journalArticle">
			<h2>Dissecting transcription regulatory pathways through a new bacterial one-hybrid reporter system</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Manman Guo</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Hui Feng</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Jun Zhang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Wenqin Wang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Yi Wang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Yuqing Li</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chunhui Gao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Huanchun Chen</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Ying Feng</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Zheng-Guo He</td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>19</td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>7</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>1301-1308</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>Genome Research</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>1088-9051</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>Jul 2009</td>
					</tr>
					<tr>
					<th>Extra</th>
						<td>PMID: 19228590
PMCID: PMC2704442</td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>Genome Res.</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1101/gr.086595.108">10.1101/gr.086595.108</a></td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>PubMed</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>eng</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>Sequence-specific DNA-binding transcription factors have 
widespread biological significance in the regulation of gene expression.
 However, in lower prokaryotes and eukaryotic metazoans, it is usually 
difficult to find transcription regulatory factors that recognize 
specific target promoters. To address this, we have developed in this 
study a new bacterial one-hybrid reporter vector system that provides a 
convenient and rapid strategy to determine the specific interaction 
between target DNA sequences and their transcription factors. Using this
 system, we have successfully determined the DNA-binding specificity of 
the transcription regulator Rv3133c to a previously reported promoter 
region of the gene Rv2031 in Mycobacterium tuberculosis. In addition, we
 have tested more than 20 promoter regions of M. tuberculosis genes 
using this approach to determine if they interact with approximately 150
 putative regulatory proteins. A variety of transcription factors are 
found to participate in the regulation of stress response and fatty acid
 metabolism, both of which comprise the core of in vivo-induced genes 
when M. tuberculosis invades macrophages. Interestingly, among the many 
new discovered potential transcription factors, the WhiB-like 
transcriptional factor WhiB3 was identified for the first time to bind 
with the promoter sequences of most in vivo-induced genes. Therefore, 
this study offers important data in the dissection of the transcription 
regulations in M. tuberculosis, and the strategy should be applicable in
 the study of DNA-binding factors in a wide range of biological 
organisms.</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2015/12/11 上午10:43:38</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2015/12/11 上午10:43:38</td>
					</tr>
				</tbody></table>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_QRACT8TF">Guo et al_2009_Dissecting transcription regulatory pathways through a new bacterial one-hybrid.pdf					</li>
					<li id="item_PBIBXSX2">PubMed entry					</li>
				</ul>
			</li>


			<li id="item_K9KB4NWC" class="item journalArticle">
			<h2>Divergent Influence to a Pathogen Invader by Resident Bacteria with Different Social Interactions</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chun-Hui Gao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Ming Zhang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Yichao Wu</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Qiaoyun Huang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Peng Cai</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="https://doi.org/10.1007/s00248-018-1207-z">https://doi.org/10.1007/s00248-018-1207-z</a></td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>77</td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>1</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>76-86</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>Microbial Ecology</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>1432-184X</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2019-01-01</td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>Microb Ecol</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1007/s00248-018-1207-z">10.1007/s00248-018-1207-z</a></td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>2019/2/19 下午4:50:21</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>Springer Link</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>en</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>Bacterial social interaction is a potential influencing factor
 in determining the fate of invading pathogens in diverse environments. 
In this study, interactions between two representative resident species 
(Bacillus subtilis and Pseudomonas putida) and a leading food-borne 
disease causative pathogen (Vibrio parahaemolyticus) were examined. An 
antagonistic effect toward V. parahaemolyticus was observed for B. 
subtilis but not for P. putida. However, the relative richness of the 
pathogen remained rather high in B. subtilis co-cultures and was, 
unexpectedly, not sensitive to the initial inoculation ratios. 
Furthermore, two approaches were found to be efficient at modulating the
 relative richness of the pathogen. (1) The addition of trace glycerol 
and manganese to Luria-Bertani medium (LBGM) reduced the richness of V. 
parahaemolyticus in the co-culture with B. subtilis and in contrast, 
increased its richness in the co-culture with P. putida, although it did
 not affect the growth of V. parahaemolyticus by its own. (2) The 
relative richness of V. parahaemolyticus on semisolid medium decreased 
significantly as a function of an agar gradient, ranging from 0 to 2%. 
Furthermore, we explored the molecular basis of bacterial interaction 
through transcriptomic analysis. In summary, we investigated the 
interactions between a pathogen invader and two resident bacteria 
species, showing that the different influences on a pathogen by 
different types of interactions can be modulated by chemicals and medium
 fluidity.</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2019/2/19 下午4:50:21</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2019/2/19 下午4:54:53</td>
					</tr>
				</tbody></table>
				<h3 class="tags">Tags:</h3>
				<ul class="tags">
					<li>课题6研究论文</li>
					<li>重点研发计划第一标注</li>
				</ul>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_N49G8887">Gao et al_2018_Divergent Influence to a Pathogen Invader by Resident Bacteria with Different.pdf						<div class="note"><div><p xmlns="http://www.w3.org/1999/xhtml" id="title"><strong>Contents</strong></p><ul xmlns="http://www.w3.org/1999/xhtml" style="list-style-type: none; padding-left:0px" id="toc"><li><a href="zotero://open-pdf/0_N49G8887/1">Abstract</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_N49G8887/1">Introduction</a></li><li style="padding-top:8px"><a href="zotero://open-pdf/0_N49G8887/2">Results</a><ul style="list-style-type: none; padding-left:12px"><li style="padding-top:4px"><a href="zotero://open-pdf/0_N49G8887/2">B. subtilis, but Not P. putida, Shows an Antagonistic Effect to V. parahaemolyticus in Broth Culture</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_N49G8887/3">V. parahaemolyticus Outcompetes B. subtilis or P. putida in Co-culture</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_N49G8887/4">Fates of V. parahaemolyticus in Co-cultures Are Significantly Affected by LBGM Medium</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_N49G8887/4">The Relative Richness of V. parahaemolyticus Is Modulated by Medium Fluidity</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_N49G8887/5">Molecular Basis of the Divergent Effects on V. parahaemolyticus by B. subtilis and P. putida</a></li></ul></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_N49G8887/7">Discussion</a></li><li style="padding-top:8px"><a href="zotero://open-pdf/0_N49G8887/9">Methods</a><ul style="list-style-type: none; padding-left:12px"><li style="padding-top:4px"><a href="zotero://open-pdf/0_N49G8887/9">Bacterial Strains and Growth Conditions</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_N49G8887/9">Colony Inhibitory Assay</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_N49G8887/9">Quantitative PCR</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_N49G8887/9">Competition Assays with Two-Species Co-cultures</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_N49G8887/9">Whole Genome Transcriptomic Analysis Using High-Throughput mRNA Sequencing</a></li></ul></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_N49G8887/10">References</a></li></ul></div>
					</div>					</li>
				</ul>
			</li>


			<li id="item_7SC4ACC4" class="item journalArticle">
			<h2>Identification and functional study of type III-A CRISPR-Cas systems in clinical isolates of Staphylococcus aureus</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Linyan Cao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chun-Hui Gao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Jiade Zhu</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Liping Zhao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Qingfa Wu</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Min Li</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Baolin Sun</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="http://www.sciencedirect.com/science/article/pii/S1438422116301084">http://www.sciencedirect.com/science/article/pii/S1438422116301084</a></td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>306</td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>8</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>686-696</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>International Journal of Medical Microbiology</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>1438-4221</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>十二月 2016</td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>International Journal of Medical Microbiology</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1016/j.ijmm.2016.08.005">10.1016/j.ijmm.2016.08.005</a></td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>2017/2/19 上午10:48:45</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>ScienceDirect</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>The CRISPR-Cas (clustered regularly interspaced short 
palindromic repeats [CRISPR]-CRISPR associated proteins [Cas]) system 
can provide prokaryote with immunity against invading mobile genetic 
elements (MGEs) such as phages and plasmids, which are the main sources 
of staphylococcal accessory genes. To date, only a few Staphylococcus 
aureus strains containing CRISPR-Cas systems have been identified, but 
no functional study in these strains has been reported. In this study, 6
 clinical isolates of S. aureus with type III-A CRISPR-Cas systems were 
identified, and whole-genome sequencing and functional study were 
conducted subsequently. Genome sequence analysis revealed a close 
linkage between the CRISPR-Cas system and the staphylococcal cassette 
chromosome mec (SCCmec) element in five strains. Comparative sequence 
analysis showed that the type III-A repeats are conserved within 
staphylococci, despite of the decreased conservation in trailer-end 
repeats. Highly homologous sequences of some spacers were identified in 
staphylococcal MGEs, and partially complementary sequences of spacers 
were mostly found in the coding strand of lytic regions in 
staphylococcal phages. Transformation experiments showed that S. aureus 
type III-A CRISPR-Cas system can specifically prevent plasmid transfer 
in a transcription-dependent manner. Base paring between crRNA and 
target sequence, the endoribonuclease, and the Csm complex were proved 
to be necessary for type III-A CRISPR-Cas immunity.</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2017/2/19 上午10:48:45</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2017/2/19 上午10:49:18</td>
					</tr>
				</tbody></table>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_NZLZ53JP">Cao et al_2016_Identification and functional study of type III-A CRISPR-Cas systems in.pdf						<div class="note"><div><p xmlns="http://www.w3.org/1999/xhtml" id="title"><strong>Contents</strong></p><ul xmlns="http://www.w3.org/1999/xhtml" style="list-style-type: none; padding-left:0px" id="toc"><li><a href="zotero://open-pdf/0_NZLZ53JP/1">1 Introduction</a></li><li style="padding-top:8px"><a href="zotero://open-pdf/0_NZLZ53JP/2">2 Results</a><ul style="list-style-type: none; padding-left:12px"><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/2">2.1 Identification of six S. aureus strains harbouring type III-A CRISPR-Cas systems from 636 clinical isolates</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/2">2.2 Whole-genome sequencing of six clinical isolates of S. aureus with type III-A CRISPR-Cas systems</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/2">2.3 CRISPR repeats in staphylococcal type III-A CRISPR-Cas systems are conserved</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/4">2.4 Comparative analysis of S. aureus CRISPR arrays</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/4">2.5 Partially complementary target sequences of CRISPR spacers have strand bias</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/5">2.6 Transcription-dependent DNA targeting of the S. aureus type III-A CRISPR-Cas immunity</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/5">2.7 The complementarity between crRNA and target in 5′ end is more important than that in 3′end</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/6">2.8 The Csm complex and Cas6 are required for antiplasmid CRISPR-Cas immunity</a></li></ul></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/6">3 Discussion</a></li><li style="padding-top:8px"><a href="zotero://open-pdf/0_NZLZ53JP/8">4 Material and methods</a><ul style="list-style-type: none; padding-left:12px"><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/8">4.1 Bacterial strains and culture conditions</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/8">4.2 Whole-genome sequencing, annotation, and bioinformatics analysis</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/9">4.3 Plasmids construction</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/9">4.4 Preparation of S. aureus electrocompetent cells and transformation experiments</a></li></ul></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/9">Competing financial interests</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/9">Nucleotide sequence accession numbers</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/9">Funding</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/9">Acknowledgements</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_NZLZ53JP/9">References</a></li></ul></div>
					</div>					</li>
				</ul>
			</li>


			<li id="item_EAD7R5HC" class="item journalArticle">
			<h2>Identification of genetic variations associated with 
epsilon-poly-lysine biosynthesis in Streptomyces albulus ZPM by genome 
sequencing</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Lin Wang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chunhui Gao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Nan Tang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Songnian Hu</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Qingfa Wu</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="http://www.nature.com/doifinder/10.1038/srep09201">http://www.nature.com/doifinder/10.1038/srep09201</a></td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>5</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>9201</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>Scientific Reports</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>2045-2322</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2015-3-17</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1038/srep09201">10.1038/srep09201</a></td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>2015/8/26 下午4:13:20</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>CrossRef</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2015/8/26 下午4:13:20</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2015/8/26 下午4:13:20</td>
					</tr>
				</tbody></table>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_DWPANZ6E">2015_Identification of genetic variations 
associated with epsilon-poly-lysine biosynthesis in Streptomyces albulus
 ZPM by genome sequencing_2.pdf						<div class="note"><div><p xmlns="http://www.w3.org/1999/xhtml" id="title"><strong>Contents</strong></p><ul xmlns="http://www.w3.org/1999/xhtml" style="list-style-type: none; padding-left:0px" id="toc"><li><a href="zotero://open-pdf/0_DWPANZ6E/1">Title</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_DWPANZ6E/4">Figure 2 Pan-genome analysis among Streptomyces species.</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_DWPANZ6E/5">Table </a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_DWPANZ6E/6">Figure 3 Identification and analysis of e-PL yield-related genetic variants.</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_DWPANZ6E/5">Table </a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_DWPANZ6E/7">Figure 4 </a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_DWPANZ6E/8">References</a></li></ul></div>
					</div>					</li>
				</ul>
			</li>


			<li id="item_FAZP8UAD" class="item journalArticle">
			<h2>InbR, a TetR family regulator, binds with isoniazid and influences multidrug resistance in Mycobacterium bovis BCG</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Min Yang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chun-Hui Gao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Jialing Hu</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Lei Zhao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Qiaoyun Huang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Zheng-Guo He</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="http://www.nature.com/doifinder/10.1038/srep13969">http://www.nature.com/doifinder/10.1038/srep13969</a></td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>5</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>13969</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>Scientific Reports</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>2045-2322</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2015-9-10</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1038/srep13969">10.1038/srep13969</a></td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>2015/9/11 下午3:02:48</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>CrossRef</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2015/9/11 下午3:02:48</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2018/1/23 上午10:11:05</td>
					</tr>
				</tbody></table>
				<h3 class="tags">Tags:</h3>
				<ul class="tags">
					<li>杨敏2016年面上项目</li>
				</ul>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_ZP3JV4WN">2015_InbR, a TetR family regulator, binds 
with isoniazid and influences multidrug resistance in Mycobacterium 
bovis BCG_2.pdf						<div class="note"><div><p xmlns="http://www.w3.org/1999/xhtml" id="title"><strong>Contents</strong></p><ul xmlns="http://www.w3.org/1999/xhtml" style="list-style-type: none; padding-left:0px" id="toc"><li><a href="zotero://open-pdf/0_ZP3JV4WN/2">Results
</a><ul style="list-style-type: none; padding-left:12px"><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/2">InbR positively regulates INH resistance in M. bovis BCG. </a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/3">InbR recognizes a palindromic motif and specifically binds to its promoter as an auto-repressor. </a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/6">InbR directly binds INH and the binding represses its DNA-binding activity. </a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/6">The function of InbR is not INH-specific and the mode of action is complicated. </a></li></ul></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/8">Discussion
</a></li><li style="padding-top:8px"><a href="zotero://open-pdf/0_ZP3JV4WN/10">Experimental Procedures
</a><ul style="list-style-type: none; padding-left:12px"><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/10">Strains, plasmids, enzymes and reagents. </a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/10">Cloning, Expression and Purification of Recombinant Proteins. </a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/12">Electrophoretic mobility shift assay (EMSA). </a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/12">ChIP-PCR and ChIP-seq assays. </a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/12">Dye primer-based DNase I footprinting assay. </a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/13">Microarray analysis. </a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/13">Quantitative real-time PCR. </a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/13">Surface plasmon resonance (SPR) analysis. </a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/13">Determination of the MIC of anti-TB drugs. </a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/13">Determination of mycobacterial growth curves and the effect of antibiotics. </a></li></ul></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/15">Acknowledgement
</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/15">Author Contributions</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/3">﻿Figure 1﻿﻿.﻿﻿ ﻿ Effect of InbR on INH resistance of M.</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/4">﻿Figure 2﻿﻿.﻿﻿ ﻿ Assays for the effects of InbR on INH resistance in M.</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/5">﻿Figure 3﻿﻿.﻿﻿ ﻿ Assays for auto-regulation of InbR.</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/7">﻿Figure 4﻿﻿.﻿﻿ ﻿ Effects of INH on the DNA-binding activity of InbR.</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/11">﻿Figure 5﻿﻿.﻿﻿ ﻿ ChIP-seq assay.</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/12">﻿Figure 6﻿﻿.﻿﻿ ﻿ qRT-PCR assays.</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/8">﻿Table 1﻿﻿. ﻿  Determination of the MIC of four anti-TB drugs.</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_ZP3JV4WN/10">﻿Table 2﻿﻿. ﻿  Expression patterns of 20 featured gene clusters in inbR-overexpressed and INH induced strains.</a></li></ul></div>
					</div>					</li>
					<li id="item_P9IF7JRK">PubMed entry					</li>
					<li id="item_S6GW6IGQ">PubMed entry					</li>
				</ul>
			</li>


			<li id="item_G6CGGCR2" class="item journalArticle">
			<h2>OxiR specifically responds to isoniazid and regulates isoniazid susceptibility in mycobacteria</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Min Yang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Li Zhang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Hui-Ling Tao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Yuan-Chao Sun</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Zhong-Zi Lou</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Wan-Zhong Jia</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Li-Hua Hu</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chun-Hui Gao</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="https://academic.oup.com/femsle/advance-article/doi/10.1093/femsle/fnz109/5498298">https://academic.oup.com/femsle/advance-article/doi/10.1093/femsle/fnz109/5498298</a></td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>FEMS Microbiology Letters</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2019-5-24</td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>FEMS Microbiol Lett</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1093/femsle/fnz109">10.1093/femsle/fnz109</a></td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>2019/5/28 下午5:03:22</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>academic.oup.com</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>en</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>Abstract.  The bacteria drug resistance is not only associated
 with the gain of drug resistance gene but also relied on the adaptation
 of bacterial cells to ant</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2019/5/28 下午5:03:22</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2019/5/28 下午5:05:01</td>
					</tr>
				</tbody></table>
			</li>


			<li id="item_XIMTJZBZ" class="item journalArticle">
			<h2>Survival of Escherichia coli O157:H7 in various soil particles: importance of the attached bacterial phenotype</h2>
				<table>
					<tbody><tr>
						<th>Type</th>
						<td>Journal Article</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Xing Liu</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Chunhui Gao</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Dandan Ji</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Sharon L. Walker</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Qiaoyun Huang</td>
					</tr>
					<tr>
						<th class="author">Author</th>
						<td>Peng Cai</td>
					</tr>
					<tr>
					<th>URL</th>
						<td><a href="http://link.springer.com/article/10.1007/s00374-016-1172-y">http://link.springer.com/article/10.1007/s00374-016-1172-y</a></td>
					</tr>
					<tr>
					<th>Volume</th>
						<td>53</td>
					</tr>
					<tr>
					<th>Issue</th>
						<td>2</td>
					</tr>
					<tr>
					<th>Pages</th>
						<td>209-219</td>
					</tr>
					<tr>
					<th>Publication</th>
						<td>Biology and Fertility of Soils</td>
					</tr>
					<tr>
					<th>ISSN</th>
						<td>0178-2762</td>
					</tr>
					<tr>
					<th>Date</th>
						<td>2017/02/01</td>
					</tr>
					<tr>
					<th>Journal Abbr</th>
						<td>Biol Fertil Soils</td>
					</tr>
					<tr>
					<th>DOI</th>
						<td><a href="http://doi.org/10.1007/s00374-016-1172-y">10.1007/s00374-016-1172-y</a></td>
					</tr>
					<tr>
					<th>Accessed</th>
						<td>2017/2/6 下午3:19:18</td>
					</tr>
					<tr>
					<th>Library Catalog</th>
						<td>link.springer.com</td>
					</tr>
					<tr>
					<th>Language</th>
						<td>en</td>
					</tr>
					<tr>
					<th>Abstract</th>
						<td>The risk of enteropathogens to food and water is highly 
dependent on their survival in soil environments. Here, the effects of 
soil type, particle size, the presence of natural organic matter (NOM) 
or Fe/Al (hydro)oxides on pathogenic Escherichia coli O157:H7 survival 
in sterilized soil particles were assessed through survival, attachment,
 metabolic activity, and qRT-PCR analyses. The abundance of inoculated 
E. coli O157:H7 in Brown soil (Alfisol) particles increased 0.6–1.4 
log10 CFU/g within 3 days (except for NOM-stripped clay), while that in 
Red soil (Ultisol) particles decreased rapidly in 8 days 
post-inoculation. Additionally, survival of bacteria was significantly 
enhanced when Fe/Al (hydro)oxides had been removed from Red soil 
particles. For the two soils, E. coli O157:H7 survived the longest in 
NOM-present clays and the bacterial adenosine 5′-triphosphate (ATP) 
levels were 0.7–2.0 times greater in clays than in sands and silts on 
day 8. Moreover, clays were more effective than silts and sands in 
binding cells and changing the expressions of acetate pathway-associated
 genes (pta and ackA). For silts and sands, E. coli O157:H7 decayed more
 rapidly in the presence of NOM and similar trends of bacterial ATP 
levels were observed between NOM-stripped and NOM-present soil 
particles, indicating that the primary role of NOM was not as a nutrient
 supply. These findings indicate that soil particles function mainly 
through attachment to change the metabolic pathway of E. coli O157:H7 
and ultimately impact the survival of bacterial pathogens in soils.</td>
					</tr>
					<tr>
					<th>Short Title</th>
						<td>Survival of Escherichia coli O157</td>
					</tr>
					<tr>
					<th>Date Added</th>
						<td>2017/2/6 下午3:19:18</td>
					</tr>
					<tr>
					<th>Modified</th>
						<td>2018/7/19 上午9:40:02</td>
					</tr>
				</tbody></table>
				<h3 class="tags">Tags:</h3>
				<ul class="tags">
					<li>VP引文</li>
					<li>课题6研究论文</li>
					<li>重点研发计划第一标注</li>
				</ul>
				<h3 class="attachments">Attachments</h3>
				<ul class="attachments">
					<li id="item_9IC6CE5B">LiuX et al_Biol Fertil Soils_2017.pdf						<div class="note"><div><p xmlns="http://www.w3.org/1999/xhtml" id="title"><strong>Contents</strong></p><ul xmlns="http://www.w3.org/1999/xhtml" style="list-style-type: none; padding-left:0px" id="toc"><li><a href="zotero://open-pdf/0_9IC6CE5B/1">Abstract</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_9IC6CE5B/1">Introduction</a></li><li style="padding-top:8px"><a href="zotero://open-pdf/0_9IC6CE5B/2">Materials and methods</a><ul style="list-style-type: none; padding-left:12px"><li style="padding-top:4px"><a href="zotero://open-pdf/0_9IC6CE5B/2">Bacterial preparation</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_9IC6CE5B/3">Characterization of soil particles</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_9IC6CE5B/3">Survival of E.coli O157:H7 in soil particles</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_9IC6CE5B/3">Attachment experiments</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_9IC6CE5B/5">Bioluminescence assay for metabolic activity via ATP determination</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_9IC6CE5B/5">qRT-PCR experiments for assessing cell response</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_9IC6CE5B/5">Statistical analysis</a></li></ul></li><li style="padding-top:8px"><a href="zotero://open-pdf/0_9IC6CE5B/5">Results</a><ul style="list-style-type: none; padding-left:12px"><li style="padding-top:4px"><a href="zotero://open-pdf/0_9IC6CE5B/5">Characterization of soil particles</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_9IC6CE5B/6">Dynamics of E.coli O157:H7 survival in soil particles</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_9IC6CE5B/6">Quantification of attachment of E.coli O157:H7 to soil particles</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_9IC6CE5B/6">Evaluation of ATP levels for E.coli O157:H7 in soil particles</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_9IC6CE5B/7">Determination of gene expression</a></li></ul></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_9IC6CE5B/7">Discussion</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_9IC6CE5B/10">Conclusions</a></li><li style="padding-top:4px"><a href="zotero://open-pdf/0_9IC6CE5B/10">References</a></li></ul></div>
					</div>					</li>
				</ul>
			</li>

		</ul>
	
</body></html>